packages/template-mrt-reference-app/app/isolation-actions.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const {LambdaClient, InvokeCommand} = require('@aws-sdk/client-lambda')
+const {S3Client, GetObjectCommand} = require('@aws-sdk/client-s3')
+const {CloudWatchLogsClient, PutLogEventsCommand} = require('@aws-sdk/client-cloudwatch-logs')
+
+export const isolationOriginLambdaTest = async (input) => {
+    const client = new LambdaClient()
+    try {
+        await client.send(new InvokeCommand(input))
+    } catch (e) {
+        if (e.name === 'AccessDeniedException') {
```

Comment:
I see this invoked down here https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1553/files#diff-5d7cbd17da0165dbb88b354d9193db13e4173e39c6488d7ceee6aba5cbcdbd80R66 with params `FunctionName` but it's unclear to me when we would catch `AccessDeniedException`. Seems it's an implementation detail of the underlying `@aws-sdk/client-lambda`'s method `InvokeCommand` but I'm unclear why we want to test that

Reply:
This is to test that the reference app isn't able to access other AWS resources, in this case, another Lambda function.